questions/questions_00

```diff
@@ -1295,163 +1250,21 @@
-80s TV: The Jeffersons was a spinoff from what show?`All In The Family 
-80s TV: The Jefferson's was a spinoff of what show?`All in the Family 
-80s TV: What 80's cartoon theme song did Ricky Martin sing?` Rubic-the Amazing Cube when he was with Menudo 
-80s TV: What 80's cartoon was a showcase for 'New Wave' Music videos?`Kidd Video 
-80s TV: What 80's group sang the theme song to Square Pegs ?`The Waitresses 
-80s TV: What 80's Pro-Wrestler was turned into a G.I. Joe character?`Sgt. Slaughter 
 80s TV: What 80's spin off of a 70's tv show did Martin Lawrence play on?`
 80s TV: What actor was famous for his character Mork from Ork`Robin Williams
-80s TV: What actor was famous for the line Nanoo, Nanoo?` Robin Williams 
-80s TV: What actress played Laura and Almonzo's niece on Little House on the Prairie?`Shannen Doherty 
-80s TV: What are the names of the Simon brothers from Simon & Simon?`Rick & AJ 
-80s TV: What cartoon featured a boy with a community of tiny people living in his wall?`The Littles 
-80s TV: What city provided the setting for One Day At A Time?`Indianapolis 
-80s TV: What current cast member of ER was on an 80's show of the same name?`George Clooney 
-80s TV: What current popular sitcom star played Michael P. Keaton's girlfriend Lauren Miller on Family Ties?`Courteney Cox 
-80s TV: What did AJ stand for in Simon & Simon's AJ Simon?`Andrew Jackson 
-80s TV: What did the P in Roscoe P. Coltrane (from Dukes of Hazzard) stand for?`Purvis 
-80s TV: What did Webster call his adoptive parents?`Ma'am and George 
-80s TV: What does ALF stand for?`Alien Life Form 
-80s TV: What does M.A.S.K. stand for?`Mobile Armoured Strike Kommand 
 80s TV: What Facts of Life character was in The Godfather?`Jo Polnyczeck's father
-80s TV: What Famous congressman was on an episode of the Golden Girls?`Sonny Bono 
-80s TV: What Georgia town did the first Dukes of Hazzard episodes take place?`Covington, Georgia 
-80s TV: What is Iola cat that died on the episode of Mamma's Family?` Midnight 
-80s TV: What island did Balki from Perfect Strangers call home?`Mypos 
-80s TV: What is MacGyver's first name?`Angus 
-80s TV: What is the name of the cat in the smurfs?`Azriel 
-80s TV: What is the name of the Dukes of Hazzards car?`General Lee 
-80s TV: What is the real name of Jem from Jem and the Holograms?`Jerrica Benton 
-80s TV: What media icon invaded Network 23?`Max Headroom 
 80s TV: What method of transportation did Webster use to get from the upper level of the house to the lower level?`The dumbwaiter in the kitchen.
-80s TV: What planet did Alf come from?`Melmac 
-80s TV: What planet did the Thundercats live on?`Third Earth. 
-80s TV: What short-lived sitcom featured Snow White and Prince Charming living as husband & wife?` The Charmings 
-80s TV: What Show did the Simpsons first appear?`The Tracy Ullman Show 
-80s TV: What show was a spin-off of Transformers?`Go-Bots 
```

Comment:
Hm, looks like the 80's TV genre doesn't follow the awesome rule you found.

Lame.

We can add these back in later I guess.